docs/_remove-global-cli.md

```diff
@@ -0,0 +1,5 @@
+> If you previously installed a global `react-native-cli` package, please remove it as it may cause unexpected issues:
+>
+> ```shell
+> npm uninstall -g react-native-cli @react-native-community/cli
```

Comment:
nit: Have you tried this by any chance? As I essentially wrote it but I haven't tested this is correct

Reply:
Yup, I've used this several times.

I didn't include `react-native` in this because I haven't seen any evidence that it's ever been recommended to globally install that package, but could to be safe.

Reply:
Awesome thank you 👍